sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/java/module-info.java

```diff
@@ -1,6 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-module com.azure.cosmos.spark {
```

Comment:
scala doesn't have support for module system:
https://github.com/scala/bug/issues/11423